client/views/actions/away.tpl

```diff
@@ -0,0 +1,7 @@
+<span role="button" class="user {{colorClass from}}" data-name="{{from}}">{{mode}}{{from}}</span>
+{{#if text}}
+went away
+<i class="away-message">({{{parse text}}})</i>
+{{else}}
+came back
+{{/if}}
```

Comment:
I think you would have to define a helper for that which would do a stupid check like `text.trim().length === 0`. Maybe @xPaw would have a better idea?